terraform/modules/data-pipelines/avl-pipeline/main.tf

```diff
@@ -89,6 +89,7 @@ module "integrated_data_avl_processor_function" {
     AVL_SUBSCRIPTION_TABLE_NAME     = var.avl_subscription_table_name
     AVL_VALIDATION_ERROR_TABLE_NAME = var.avl_validation_error_table_name
     GTFS_TRIP_MAPS_TABLE_NAME       = var.gtfs_trip_maps_table_name
+    ENABLE_CANCELLATIONS            = var.enable_cancellations
```

Comment:
I think terraform handles this but I'd maybe make the env var a string explicitly `var.enable_cancellations ? "true" : "false"`